content/press/press-content.md

```diff
@@ -1,32 +1,19 @@
 <!--This is the main content file to edit for this page. It is embedded in src/pages/press.js-->
 
-## Press: Contact environmentalenforcementwatch@gmail.com
+**Press Contact: environmentalenforcementwatch@gmail.com
 
 <!--Remove '<!--' and similar-looking closing tag below to uncomment when we have some coverage -->
 <!--# Press Coverage of Congressional Committee Reports-->
 
-## Press Coverage of COVID EPA Enforcement Report
+## EEW Reports
 
-IMAGE = REPORT COVER
+* Executive Summary – 2020 EEW Congressional Report Card Findings (no link yet)
+* [More Permission to Pollute: The Decline of EPA Enforcement and Industry Compliance during COVID (10/13/2020)](https://envirodatagov.org/more-permission-to-pollute-the-decline-of-epa-enforcement-and-industry-compliance-during-covid/) 
```

Comment:
In this new format, do we no longer need a cover image?

Reply:
I don't think we do